lib/hyperion/result_handling/result_maker.rb

```diff
@@ -35,6 +36,8 @@ def make_from_typho(typho_result, result_class)
       read_body = proc { read(typho_result.body, route.response_descriptor) }
       code = typho_result.code
 
+      log_error_response(read(typho_result.body, :json)) unless typho_result.success?
```

Comment:
To be more concrete: in this particular case, `ResultMaker` was factored out of `Hyperion` because making results is at a lower level of detail than the rest of the code in `Hyperion`. Does this mean that now that we've factored it out, we must add tests for this private functionality? No.

The process I'd use for adding tests in this situation is: is there a corresponding spec file? No. Hmm, if tests don't use this code, what does? Use rubymine to Find Usages. Ok, it's only used by `Hyperion`. Does `Hyperion` have tests? Yes. Are there tests that concern `ResultMaker`? Yes.

As with most private code, we don't test `ResultMaker` directly because doing so would overconstrain the set of valid implementations of `Hyperion`, leading to brittle tests and more difficult maintenance.